### PR TITLE
display ice production

### PR DESCRIPTION
**Pull request by BreadlinSan**

### CODE REVIEW

File: src/app/pages/owner/owner.page.ts [hunk 1]
```diff
@@ -8,14 +8,16 @@ import { CommonService } from 'src/app/services/api/common.service';
   styleUrls: ['./owner.page.scss'],
 })
 export class OwnerPage implements OnInit {
-
+  workers: ISale[];
```
Comment (herbertbruno) [inline src/app/pages/owner/owner.page.ts]: Why are you getting this as ISale type ?

Comment (herbertbruno) [reply src/app/pages/owner/owner.page.ts]: @breadl

File: src/app/interfaces/sale.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export interface ISale {
+    iceProduced: number;
```
Comment (herbertbruno) [inline src/app/interfaces/sale.ts]: Why did you add this?

File: src/app/pages/owner/owner.page.ts [hunk 3]
```diff
@@ -33,7 +35,18 @@ export class OwnerPage implements OnInit {
       }
       this.waitingFlag = false;
     })
-  }
 
+    this.waitingFlag = true;
+
+    this.commonSer.getCollectionList('worker').then(cloudSalesData => {
+      this.workers = cloudSalesData;
```
Comment (herbertbruno) [inline src/app/pages/owner/owner.page.ts]: rename cloudSalesData to cloudWorkerProductionData. it will work but good to give correct name

**Comment by BreadlinSan** [comment]
I couldn't get about this comment.

**Comment by BreadlinSan** [comment]
Okk
